var/spack/repos/builtin/packages/libsonata-report/package.py

```diff
@@ -16,7 +16,7 @@ class LibsonataReport(CMakePackage):
     homepage = "https://github.com/BlueBrain/libsonata"
     git = "https://github.com/BlueBrain/libsonata.git"
 
-    version('0.1.1', commit='1f1c7cbe1bc674', submodules=True, get_full_repo=True)
+    version('0.1.2', commit='c72da93e0dda72', submodules=True, get_full_repo=True)
```

Comment:
no tags?

Reply:
No tags since the writers were not accepted in libsonata, so for now, while we find a place is just a commit in the branch reports

Reply:
Alright… I approved anyways, so feel free to merge.

Reply:
Waiting for the simulationStack CI to finish....
https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.SimulationStack/detail/hpc.SimulationStack/2460/pipeline

Reply:
:facepalm: yeah, I didn't look at that